TheMET/TheMET/FeaturedArtViewController.swift

```diff
@@ -11,4 +11,13 @@ import UIKit
 
 class FeaturedArtViewController: UIViewController {
     
+    override func viewDidLoad() {
+           super.viewDidLoad()
+           
+        let newView = FeaturedArtWorkLoadingView.constructView()
+        newView.awakeFromNib()
+        self.view.addSubview(newView)
+        
```

Comment:
`newView` is added to controller's view. But the constraints are not specified. Pls setup top, bottom, leading and trailing constraints of this view.

Reply:
fixed